### PR TITLE
Adds Tribadism.

### PR DESCRIPTION
**Pull request by pireamaineach**
God wills it.

trib·a·dism
/ˈtribəˌdizəm/
noun
a lesbian practice in which one partner lies on top of the other and simulates the movements of the male in heterosexual intercourse.

:cl:
add: Adds a new E.R.P. verb.
/:cl:

### CODE REVIEW

**Comment by pireamaineach** [comment]
Am I-- wait fuck I made that P.R. with my master didn't I.

**Comment by pireamaineach** [comment]
I am not smart.

**Comment by pireamaineach** [comment]
All is right with the universe.

File: interactions/lewd/lewd_datums.dm [hunk 1]
```diff
@@ -245,6 +245,20 @@
...
+	interaction_sound = null
+	require_target_vagina = TRUE
+	require_user_vagina = TRUE
+	user_not_tired = TRUE
+	require_user_bottomless = TRUE
+	require_target_bottomless = TRUE
+	max_distance = 0
+
 /datum/interaction/lewd/mountass/display_interaction(mob/living/carbon/human/user, mob/living/carbon/human/target)
 	user.do_mountass(target)
```
Comment (LetterN) [inline interactions/lewd/lewd_datums.dm]: add the code here not in the middle of `mountass/display_interaction(...`

Comment (pireamaineach) [reply interactions/lewd/lewd_datums.dm]: Is this what you meant?

Comment (LetterN) [reply interactions/lewd/lewd_datums.dm]: yes, put the code you added bellow the `user.do_mountass(target)`

Comment (pireamaineach) [reply interactions/lewd/lewd_datums.dm]: Done.

**Pull request merged**